src/xmipp/libraries/data/symmetries.cpp

```diff
@@ -2437,7 +2437,7 @@ void symmetry_Dihedral(MultidimArray<double> &Vout, const MultidimArray<double>
 			}
 			z+=zStep;
```

Comment:
z is increased twice

Reply:
This comment brings me to life! I've been looking for the bug that makes fail the test Thank you so much!! Now let's see if the tests are fixed